Table scrolling

LostPaul opened the pull request:
Showcase:
https://user-images.githubusercontent.com/70213368/217935370-00f4cf50-de12-4e37-b8e7-211fb2859f42.mp4

I also added a option to disable the "Edit this block" button next to tables.

LostPaul committed:
Added scrolling to Tables

LostPaul committed:
Added the !important back

Stardusten commented:
Perhaps moving the "Edit this block" button to the left would be a better option? Like the minimal theme does.

![image](https://user-images.githubusercontent.com/38722307/217964190-f631d182-2ba3-4e28-adb7-df629b7cb7db.png)

LostPaul commented:
> Perhaps moving the "Edit this block" button to the left would be a better option? Like the minimal theme does.
> 
> ![image](https://user-images.githubusercontent.com/38722307/217964190-f631d182-2ba3-4e28-adb7-df629b7cb7db.png)

Good idea, I tried that but it looks like the theme modifies the whole editor that it can fit like that but there's probably a way to display the button like that without modifying the content container, maybe something like a different z-index but I'm not really an front-end dev so I don't wanna accidentally mess something up you can add that though if you want to.

LostPaul committed:
Merge branch 'v2' into v2

LostPaul committed:
Replace newline with br when pasting

LostPaul committed:
Replace newline with br when pasting

LostPaul committed:
Revert "Replace newline with br when pasting"

This reverts commit a2a5c63c783a3e4ee234cd2388961fa379187d16.

LostPaul commented:
oops I didn't mean to add the commit here

LostPaul committed:
nvm way better option to do it

LostPaul committed:
ReplaceAll instead of only replace

LostPaul committed:
Instead of the end start at the beginning

Before that change when you clicked on a table cell it set the cursor at the end of the cell.
The newline changes the size of the cell that results in the text to be longer than what the users see and the cursor goes out of index and it fails to set the cursor in the cell.

LostPaul committed:
Update mousedownHandler.ts

LostPaul committed:
Merge pull request #1 from LostPaul/Fix-for-bold-and-other-things-break-clicking-table-cells-#58-

Update mousedownHandler.ts

LostPaul committed:
Merge pull request #2 from LostPaul/Replace-newline-with-br-when-pasting

Replace newline with br when pasting
